deps/rabbit/src/rabbit_db.erl

```diff
@@ -102,6 +107,23 @@ init_using_khepri() ->
                #{domain => ?RMQLOG_DOMAIN_DB})
     end.
 
+init_finished() ->
+    %% Used during initialisation by rabbit_logger_exchange_h.erl
+    %% If an exchange logger is configured, it needs to declare the
+    %% exchange. For this, it requires the metadata store to be
+    %% initialised. The initialisation happens on a rabbit boot step,
+    %% after the second phase of the prelaunch where the logger is
+    %% configured.
+    %% Using this persistent term the logger exchange can delay
+    %% declaring the exchange until the metadata store is ready.
+    persistent_term:put(?PT_KEY_INIT_FINISHED, true).
+
+is_init_finished() ->
+    persistent_term:get(?PT_KEY_INIT_FINISHED, false).
+
+clear_init_finished() ->
+    persistent_term:erase(?PT_KEY_INIT_FINISHED).
```

Comment:
We probably don't need to return the return value of `persistent_term:erase/1`.

Can you ignore the return value and always return `ok`?